fronts-client/src/components/FrontsEdit/Collection.tsx

```diff
@@ -114,6 +114,7 @@ interface ConnectedCollectionContextProps extends CollectionContextProps {
 	handleBlur: () => void;
 	lastDesktopArticle?: string;
 	lastMobileArticle?: string;
+	groupsIds: string[];
```

Comment:
Nit : sorry, could this be renamed to groupIds rather than groupsIds?

Reply:
Yep, that's better - done!